### ISSUE TITLE
Moving "Find result" Window is Laggy when using a High Mouse Polling Rate (>125Hz)

### ISSUE THREAD

**Issue by mushnamukk** [notepad-plus-plus/notepad-plus-plus#3529]
### Description of the Issue

Moving "Find result" Window is Laggy when using a High Mouse Polling Rate (>125Hz)
This exact problem was present in all Microsoft Office programs (2013 & 2016 Versions) and was only recently fixed. 

Here is a link to a reddit thread where the original poster mentions the issue and the top commenter figures out the problem:
https://www.reddit.com/r/Windows10/comments/5fkyxt/windows_10_lag_with_word_window_dragging/

*Note: I am using Windows 10 1703 Build 15063.483, can confirm that this issue has existed for as long as I have had Windows 10. Unsure if this bug exists in Windows 7 or 8 (or any other OS, for that matter).

### Steps to Reproduce the Issue

1. You must have a mouse that supports software to change the polling rate/frequency. This is common on all gaming mice. Set the polling rate to 1ms/2ms (or 1000MHz/500MHz depending on the mouse software interface and the highest frequency your mouse supports)
2. Open the 'Find results' window, either using 'Search>Search Results Window' or using F7 in Notepad++ on Windows
3. Try dragging the window around

### Expected Behavior

Fluid and responsive movement of the window

### Actual Behavior

Very laggy, unresponsive, and inconsistent redrawing and movement of the window

**Comment by josh08287** [comment]
This also affects the document map, it seems as though anything that creates a new scintilla window is affected by this. See issue 3350 https://github.com/notepad-plus-plus/notepad-plus-plus/issues/3550.

**Comment by fusionneur** [comment]
Same issue on 7.5.1

**Comment by mushnamukk** [comment]
Can anyone confirm if this bug is exclusive to Windows 10?

**Comment by josh08287** [comment]
No, this issue is present in Windows 7 as well. It affects any dockable dialog.

**Comment by mushnamukk** [comment]
This is beginning to make me think that the problem is deep-rooted into the Windows OS. I am beginning to doubt this bug was even fixed in Microsoft Office because I believe I am running the latest build of Office 2016 Professional and the problem persists. I had the same problem with 2013 and thought upgrading might fix it but it did not.

If you have access to Word, Excel, etc. could you see if the problem exists when dragging those windows around? 

Please include your version of Office if you can test this. 

Thanks.

**Comment by josh08287** [comment]
I have no problems with Office (2016 Professional). This issue still exists in 7.5.1 in both x86 and x64 with a high polling rate mouse. 
It appears that the polling rate causing the OnMouseMove event firing more rapidly is the culprit, and the code within the event handler is taking longer than it should to return to the compositer.

**Comment by seeker25** [comment]
Having the same issue. It literally just craps out my computer and prevents the mouse from moving at all.

**Comment by ArkadiuszMichalski** [comment]
Next report: https://community.notepad-plus-plus.org/topic/22159/dragging-find-result-window-is-very-slow/6
